neuland/digitizing/R3BDigitizingPaddleNeuland.cxx

```diff
@@ -85,15 +96,16 @@ namespace R3B::Digitizing::Neuland
         return { leftChannelHit, rightChannelHit };
     }
 
-    auto NeulandPaddle::GenerateChannelHit(const Double_t mcTime, const Double_t mcLight, const Double_t dist)
-        -> Channel::Hit
+    auto NeulandPaddle::GenerateChannelHit(const double mcTime, const double mcLight, const double dist) const
+        -> const Channel::Hit
     {
-        auto time = mcTime + (NeulandPaddle::gHalfLength - dist) / DEFAULT_EFFECTIVE_C;
-        auto light = mcLight * std::exp(-NeulandPaddle::gAttenuation * (NeulandPaddle::gHalfLength - dist));
+        auto time = mcTime + (NeulandPaddle::gHalfLength_ - dist) / effective_speed_;
+        auto light = mcLight * std::exp(-NeulandPaddle::gAttenuation_ * (NeulandPaddle::gHalfLength_ - dist));
```

Comment:
Here the `time` and `light` values are also depending on `time_offset_` and `time_sync_` parameters.